pages/common/expr.md

```diff
@@ -3,22 +3,30 @@
 > Evaluate expressions and manipulate strings.
 > More information: <https://www.gnu.org/software/coreutils/expr>.
 
-- Get string length:
+- Get a specific string length:
```

Comment:
How about:
```suggestion
- Get the length of a specific string:
```

Reply:
I personally don't like it due to `the length of`. As I understand it is preferred to use `a`/`an` articles instead of `the` to make descriptions more general. That's why I refresh many pages now to standardize them. 😄

Reply:
`the length of` makes more sense in this context.

Reply:
I don't know why it makes more sense but it maybe more readable. 😄